build.yaml

```diff
@@ -508,6 +508,36 @@ steps:
    publishAs: batch
    dependsOn:
      - service_base_image
+ - kind: runImage
+   name: render_grafana_nginx_conf
+   image:
+     valueFrom: service_base_image.image
+   script: |
+     set -ex
+     cd /io
+     rm -rf repo
+     mkdir repo
+     cd repo
+     {{ code.checkout_script }}
+     cd grafana
+     python3 ../ci/jinja2_render.py '{"deploy": false, "default_ns": {"name": "{{ default_ns.name }}"}}' nginx.conf nginx.conf.out
```

Comment:
@danking Here's what I think a jinja templating step would consist of. I feel like this is also not quite as straightforward as I would have liked because there's two template rendering steps happening, first the bash script is rendered by CI and then the actual job does rendering of the conf file. `false` is not correct here but I was having trouble with `{{ deploy }}` since CI's rendering would fill in `False` instead of `false`, so `jinja2_render.py` threw a JSON decoding error. Any thoughts?

Reply:
Could maybe do `"default_ns": {{ default_ns }}` but idk if it renders as a json object. Either way that doesn't solve the deploy awkwardness